tests/tests/test_brainmapper/test_analysis.py

```diff
@@ -1,33 +1,92 @@
 from pathlib import Path
 
+import numpy as np
 import pandas as pd
 import pytest
+from brainglobe_atlasapi import BrainGlobeAtlas
 
 from brainglobe_utils.brainmapper.analysis import (
     Point,
     count_points_per_brain_region,
+    summarise_points_by_atlas_region,
 )
 
 
 @pytest.fixture
 def points():
     p1 = Point(
-        raw_coordinate=[1.0, 1.0, 1.0],
-        atlas_coordinate=[1.0, 1.0, 1.0],
+        raw_coordinate=np.array([1.0, 1.0, 1.0]),
+        atlas_coordinate=np.array([1.0, 1.0, 1.0]),
         structure="Paraventricular hypothalamic nucleus, descending division",
         structure_id=56,
         hemisphere="right",
     )
     p2 = Point(
-        raw_coordinate=[2.0, 2.0, 2.0],
-        atlas_coordinate=[2.0, 2.0, 2.0],
+        raw_coordinate=np.array([2.0, 2.0, 2.0]),
+        atlas_coordinate=np.array([2.0, 2.0, 2.0]),
         structure="Anterodorsal nucleus",
         structure_id=57,
         hemisphere="left",
     )
     return [p1, p2]
 
 
+@pytest.fixture
+def points_mouse():
+    """
+    List of points in different regions of the allen mouse brain atlas 100um.
+    All values are correct except for the raw_coordinate, which is set to an
+    arbitrary value.
+    """
+    return [
```

Comment:
Question for my own understanding here - why does this fixture return a list (I noticed the `points` fixture also returns a list? We could just return the points using the normal Python syntax right - unless we want to convert it to an array (in which case we could return the array)?

Unless there's some pytest fixture law that says they can't return multiple values?

Reply:
So I just checked, and it does work with the normal Python syntax for multiple return values. I think it's returning a list here, as the functions that are being called with points e.g. [`count_points_per_brain_region`](https://github.com/brainglobe/brainglobe-utils/blob/main/brainglobe_utils/brainmapper/analysis.py#L162) specify list in the docstrings. PyCharm complains a bit when given a tuple instead, underlining variables with yellow.

Would it be best to use the normal python syntax then convert to list inside the function? Or just ignore PyCharm complaining about tuple vs list?

Reply:
PyCharm is likely complaining because it's doing some clever type-checking under the hood. AFAIK returning a tuple using the usual return comma-separated syntax (`return out1, out2, ` etc) is the standard Python way.

But I don't think it's worth dying on this hill here and converting list -> tuple, I was just curious about the design choice :grin: